### PR TITLE
Fixes #474 :Developed Contributors' Section

### LINKED ISSUE #474: Dead end button. And add a button

### PR DESCRIPTION
**Pull request by saarthakchats**
Fixes #474

### CODE REVIEW

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Provide Test/Review Link

**Comment by saarthakchats** [comment]
https://saarthakchats.github.io/gci17.fossaaaaaasia.org/
@0x48piraj

**Review by 0x48piraj — APPROVED** [review]
Looks good.

File: index.html [hunk 1]
```diff
@@ -109,7 +109,8 @@ <h2 class="animated fadeIn">Contributors</h2>
 		<div class="col-sm-12 col-md-12 text-center spread-children-large mentor-list" id="contributors"></div>
 		
 		<div class="col-sm-12 col-md-12 text-center">
-	<a  class="btn" data-toggle="popover" title="" data-placement="bottom" data-content="<a href='https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia/'>Click Here</a>" data-original-title="Check Out the starter best practices">How can I be a contributor?</a>	
+	<a  class="btn" data-toggle="popover" title="" data-placement="bottom" href="https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia/"> How can I be a contributor?</a>	
+	<a  class="btn" data-toggle="popover" title="" data-placement="bottom" href="https://github.com/gci17.fossasia.org">View the Github Repository <i class="fa fa-github fa-lg"></i></a>
```
Comment (CloudyPadmal) [inline index.html]: Wrong link : https://github.com/fossasia/gci17.fossasia.org

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -109,7 +109,8 @@ <h2 class="animated fadeIn">Contributors</h2>
 		<div class="col-sm-12 col-md-12 text-center spread-children-large mentor-list" id="contributors"></div>
 		
 		<div class="col-sm-12 col-md-12 text-center">
-	<a  class="btn" data-toggle="popover" title="" data-placement="bottom" data-content="<a href='https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia/'>Click Here</a>" data-original-title="Check Out the starter best practices">How can I be a contributor?</a>	
+	<a  class="btn" data-toggle="popover" title="" data-placement="bottom" href="https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia/"> How can I be a contributor?</a>
```
Comment (CloudyPadmal) [inline index.html]: And remove the space in front of `How`

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]

**Comment by saarthakchats** [comment]
@CloudyPadmal

**Comment by anshumanv** [comment]
What did you exactly do?
Please don't remove the PR template and fill it appropriately.

**Review by anshumanv — APPROVED** [review]

**Comment by saarthakchats** [comment]
The button "How can I contribute" leads nowhere. I have fixed that and also added a button to the github repos. @anshumanv

**Comment by saarthakchats** [comment]
I have fixed a button and added another useful one

**Review by gabru-md — APPROVED** [review]

**Comment by saarthakchats** [comment]
@anshumanv @gabru-md 
If you think it is fine, can you approve my task as well https://codein.withgoogle.com/dashboard/task-instances/6443522922119168/

**Comment by saarthakchats** [comment]
@gabru-md @anshumanv Can we merge. 
No pressure ;)

**Review by CloudyPadmal — APPROVED** [review]

**Comment by saarthakchats** [comment]
@gabru-md Please approve my task

**Comment by saarthakchats** [comment]
https://codein.withgoogle.com/dashboard/task-instances/6443522922119168/

**Pull request merged**